cmd/podman/cp.go

```diff
@@ -0,0 +1,366 @@
+package main
+
+import (
+	"archive/tar"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"syscall"
+
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/pkg/chrootuser"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/archive"
+	"github.com/containers/storage/pkg/chrootarchive"
+	"github.com/containers/storage/pkg/idtools"
+	digest "github.com/opencontainers/go-digest"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	cpDescription = "Copy files/folders between a container and the local filesystem"
+	cpCommand     = cli.Command{
+		Name:         "cp",
+		Usage:        "Copy files/folders between a container and the local filesystem",
+		Description:  cpDescription,
+		Action:       cpCmd,
+		ArgsUsage:    "CONTAINER:SRC_PATH DEST_PATH | SRC_PATH|- CONTAINER:DEST_PAT",
```

Comment:
[CONTAINER:]SRC_PATH [CONTAINER:]DEST_PATH

(This loses the nuance that only one can be a container, but I think it's still a lot clearer)